src/main/preload.js

```diff
@@ -0,0 +1,11 @@
+// eslint does not allow me to put electron in devDependencies
+// electron-builder does not allow me to put electron in dependencies
+// eslint-disable-next-line import/no-extraneous-dependencies
+const { ipcRenderer } = require('electron');
+const { EventEmitter } = require('events');
+
+window.electron = {
```

Comment:
isn't it `global` on the main process ?

Reply:
>>> preload String (optional) - Specifies a script that will be loaded before other scripts run in the page. This script will always have access to node APIs no matter whether node integration is turned on or off. The value should be the absolute file path to the script. When node integration is turned off, the preload script can reintroduce Node global symbols back to the global scope. See example here.

https://electronjs.org/docs/api/browser-window